packages/ai-preact/tests/mcp-apps.test.tsx

```
@@ -0,0 +1,205 @@
+// renders fine, but in tests we only want to inspect the wired callbacks.
+let capturedProps: Record<string, unknown> = {}
+
+vi.mock('@mcp-ui/client', () => ({
+  AppRenderer: (props: Record<string, unknown>) => {
+    capturedProps = props
+    return null
+  },
+}))
+
+// Import AFTER mock is registered.
+import { MCPAppResource } from '../src/mcp-apps'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)